chips/stm32f303xc/src/wdt.rs

```diff
@@ -81,33 +80,10 @@ impl WindoWdg {
         self.enabled.set(true);
     }
 
-    pub fn set_client(&self, client: &'static dyn kernel::watchdog::WatchdogClient) {
-        self.client.set(client);
-    }
-
-    pub fn handle_interrupt(&self) {
-        if self.registers.sr.is_set(Status::EWIF) {
-            self.registers.sr.modify(Status::EWIF::CLEAR);
-
-            // We are feeding the watchdog here to avoid getting a reset
-            // as a consequence of staying in the sleep state for too long.
-            // To get a better idea, check the kernel_loop() function
-            // in kernel/src/sched.rs and the sleep() function in ./chip.rs.
-            unsafe {
-                if WATCHDOG_SLEEP_FLAG {
-                    self.tickle();
-                    WATCHDOG_SLEEP_FLAG = false;
-                } else {
-                    self.client.map(|client| client.reset_happened());
-                }
-            }
-        }
-    }
-
     /// This interrupt is only cleared by hardware after a reset.
-    fn enable_interrupt(&self) {
-        self.registers.cfr.modify(Config::EWI::SET);
-    }
+    // fn enable_interrupt(&self) {
+    //     self.registers.cfr.modify(Config::EWI::SET);
+    // }
```

Comment:
Where is this interrupt enabled now?

Reply:
I was intending to delete this, as I am not sure what to do when we receive the interrupt (if we activate it at all).

Reply:
So the reset still happens even if the interrupt is disabled?

Reply:
yes